<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>参数设置</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>571</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>路径设计参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>路径图样：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="RouteType">
           <item>
            <property name="text">
             <string>普通树形路径</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>XY正交树形</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>环形路径</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>S型路径</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>是否隔行：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="IF_Interlaced">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>从内向外：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="IF_I2O">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>树形筋间距：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="TreeVeinDif">
           <property name="value">
            <double>0.300000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>121</y>
       <width>491</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>角度薄片参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_14">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>薄片厚度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sheetThickness">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>薄片高度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sheetHeight">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>薄片宽度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sheetWidth">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_15">
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>基台层数：</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="LayerSliceThickness">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>20</y>
           <width>58</width>
           <height>20</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>20</y>
           <width>84</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>激光切片厚度：</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="BaseLayerNum">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>531</width>
       <height>52</height>
      </rect>
     </property>
     <property name="title">
      <string>填充参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>拓展轮廓间距：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ExtendedContourFillSpace">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>轮廓与填充间距：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ContourFillSpace">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>填充间距：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="FillSpace">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>534</width>
       <height>164</height>
      </rect>
     </property>
     <property name="title">
      <string>功率参数</string>
     </property>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>521</width>
        <height>40</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>扩展轮廓功率：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>10</y>
         <width>108</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>扩展轮廓扫描速度：</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ExtendedContourSpeed">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ExtendedContourPower">
       <property name="geometry">
        <rect>
         <x>137</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>68</y>
        <width>521</width>
        <height>40</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>轮廓功率：</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ContourPower">
       <property name="geometry">
        <rect>
         <x>137</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>10</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>轮廓扫描速度：</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ContourSpeed">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>114</y>
        <width>521</width>
        <height>40</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>填充功率：</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="FillPower">
       <property name="geometry">
        <rect>
         <x>137</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>264</x>
         <y>10</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>填充扫描速率：</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="FillSpeed">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>58</width>
         <height>20</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>531</width>
       <height>52</height>
      </rect>
     </property>
     <property name="title">
      <string>薄片位置参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>薄片倾斜角度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="TiltAngle">
        <property name="maximum">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>薄片旋转角度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="RotateAngle">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>薄片中心位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="BaseMidPointX">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="BaseMidPointY">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>361</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>图像演示</string>
    </property>
    <widget class="QPushButton" name="DrawButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始绘图</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>341</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_13">
        <property name="title">
         <string>切片层数</string>
        </property>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>22</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>当前层数：</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>331</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actOpen"/>
    <addaction name="actNew"/>
    <addaction name="actSave"/>
    <addaction name="actSaveOtherPlace"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actNew"/>
   <addaction name="actSave"/>
   <addaction name="actSaveOtherPlace"/>
   <addaction name="actionDrawbut"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/images/new2.bmp</normaloff>:/images/new2.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/images/open3.bmp</normaloff>:/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开配置文件</string>
   </property>
  </action>
  <action name="actSaveOtherPlace">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/images/120.bmp</normaloff>:/images/120.bmp</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/images/104.bmp</normaloff>:/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionDrawbut">
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/images/20.ico</normaloff>:/images/20.ico</iconset>
   </property>
   <property name="text">
    <string>Drawbut</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
